operator-framework/src/main/java/io/javaoperatorsdk/operator/api/UpdateControl.java

```diff
@@ -27,6 +27,15 @@ private UpdateControl(
     return new UpdateControl<>(customResource, true, false);
   }
 
+  /**
+   * As a results of this there will be two call to K8S API. First the custom resource will be
+   * updates then the status sub-resource.
+   */
+  public static <T extends CustomResource>
```

Comment:
how about `updateCustomResourceAndStatus`?

Reply:
hmm, makes sense will change it!! thx